src/test/java/codeu/model/store/basic/ActivityStoreTest.java

```diff
@@ -0,0 +1,101 @@
+package codeu.model.store.basic;
+
+import codeu.model.data.Action;
+import codeu.model.data.Activity;
+import static codeu.model.data.ModelDataTestHelpers.assertActivityEquals;
```

Comment:
Why is only this import static?

Reply:
because I specifically exported the function "assertActivityEquals". When you export a function, you put static next to the import

Reply:
That's interesting! I have never tried to only export one method, cool!